File: yazi/.config/yazi/flavors/eldritch.yazi/.theme/theme.php
<?php
// comment.line.double-slash
# comment.line.number-sign
/* comment.block */

/**
 * comment.documentation
 */

define('NUM', 42);                    // constant.numeric
define('ESC', "\n");                  // constant.character.escape
define('FLAG', true);                 // constant.language

class MyClass extends BaseClass {      // entity.name.type, entity.inherited-class
    public function myFunction($param) { // entity.name.function, variable.parameter
        $color = "#ff0000";           // constant.other, string.quoted.double
        if (FLAG) {                   // keyword.control
            return $color;            // keyword.control
        }
        return "";
    }
}
?>

File: yazi/.config/yazi/flavors/eldritch.yazi/.theme/theme.rb
# comment.line.number-sign
=begin
comment.block
=end

NUM = 42                         # constant.numeric
ESC = "\n"                       # constant.character.escape
FLAG = true                      # constant.language

class MyClass < BaseClass        # entity.name.type, entity.inherited-class
    def my_function(param)       # entity.name.function, variable.parameter
        color = "#ff0000"        # constant.other, string.quoted.double
        if FLAG                  # keyword.control
            return color         # keyword.control
        end
    end
end
